tracer/src/Datadog.Trace/PlatformHelpers/AspNetCoreHttpRequestHandler.cs

```diff
@@ -118,8 +120,12 @@ public Scope StartAspNetCorePipelineScope(Tracer tracer, HttpContext httpContext
             }
 
             var scope = tracer.StartActiveInternal(_requestInOperationName, propagatedContext, tags: tags);
-
-            scope.Span.DecorateWebServerSpan(resourceName, httpMethod, host, url, tags, tagsFromHeaders);
+            scope.Span.DecorateWebServerSpan(resourceName, httpMethod, host, url, userAgent, tags, tagsFromHeaders);
+            var peerIp = new Headers.Ip.IpInfo(httpContext.Connection.RemoteIpAddress?.ToString(), httpContext.Connection.RemotePort);
+            if (!tracer.Settings.IpHeaderDisabled)
+            {
+                Headers.Ip.RequestIpExtractor.AddIpToTags(peerIp, request.IsHttps, key => request.Headers.TryGetValue(key, out var value) ? value : string.Empty, tracer.Settings.IpHeader, tags);
```

Comment:
Ultra NIT for grandpa - naming the last but one argument would have helped me, even maybe puting the func in a var the line before for readability.

Reply:
fixed in 7e9bb09d8d319eee8ace969f97cdeedb50c022b5